src/CommandingOfficers/Modifiers/COModifier.java

```diff
@@ -1,9 +1,31 @@
 package CommandingOfficers.Modifiers;
 
 import CommandingOfficers.Commander;
+import Engine.Combat.CombatParameters;
 
-public interface COModifier
+// COModifier provides static bonuses to units,
```

Comment:
You should use Javadoc comments for class/function/variable descriptions.
This allows Eclipse to show the comment when you hover over it from somewhere else.